### PR TITLE
Improved about section.

### PR DESCRIPTION
**Pull request by hsorby**
If someone (who isn't me, or probably also @kerimoyle) could check the state of staging and then give this the ok or decline.

### CODE REVIEW

**Comment by agarny** [comment]
- I would definitely remove the sentence that reads "The library is intended to be lightweight and focused purely in dealing with CellML models. In achieving this, some of the convenience methods/objects that we might expect will not be part of libCellML." I find this kind of sentence unnecessary, not to say that it wouldn't make me want to use libCellML.
-  I would drop the "www" in "... link to the official libCellML website www.libcellml.org..."
- Might want a full stop at the end of the CellML 2.0 paper citation?
- Our copyright currently reads "Copyright © 2021 libCellML", but libCellML clearly started before 2021. Shouldn't we list the year we really started working on libCellML, i.e. 2015?

**Comment by hsorby** [comment]
With regards to copyright my understanding is that only the last date is of any relevance.  Also no quite sure why we are copyrighting this work when we are really more cc by 4.0.

**Comment by nickerso** [comment]
I agree that we can drop the copyright and just make sure everything has an appropriate license.

and apart from those other suggestions from @agarny , this looks good to merge into production.

File: src/App.vue [hunk 1]
```diff
@@ -60,7 +60,14 @@
...
         <v-col class="text-center">
-          <p id="footer-copyright">Copyright &#169; 2021 libCellML</p>
+          <p id="footer-copyright">
+            <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"
+              ><img
+                alt="Creative Commons Licence"
+                style="border-width:0"
+                src="https://i.creativecommons.org/l/by/4.0/88x31.png"
+            /></a>
+          </p>
```
Comment (agarny) [inline src/App.vue:70]: So, now, we can see the license twice when we scroll down to the bottom of the page. Odd.

File: src/components/AboutContent.vue [hunk 2]
```diff
@@ -6,10 +6,8 @@
 
         <h3>Questions and comments</h3>
         <p>
-          The library is intended to be lightweight and focused purely in
-          dealing with CellML models. In achieving this, some of the convenience
-          methods/objects that we might expect will not be part of libCellML.
-          For questions, feature requests, and bug reports about the libCellML
+          The library is designed to be used for working with CellML models. For
+          questions, feature requests, and bug reports about the libCellML
```
Comment (agarny) [inline src/components/AboutContent.vue:10]: Minor: we really ought to have one sentence per line. Makes it much easier to review changes.

File: src/components/AboutContent.vue [hunk 3]
```diff
@@ -63,7 +59,7 @@
           >
             our documentation</a
           >
-          are shown below.
+          are shown below (in an actively randomised order).
```
Comment (agarny) [inline src/components/AboutContent.vue]: I would definitely remove that parenthesis comment. People don't (shouldn't have to) care about this.

File: src/components/GithubContributors.vue [hunk 4]
```diff
@@ -17,12 +17,14 @@
...
-                <v-col class="avatar_name">{{ person.name }}</v-col>
+                <v-col class="avatar_name">{{
+                  person.name || person.login
+                }}</v-col>
               </v-row>
             </a>
           </template>
           <span>
-            <strong>{{ person.name }}</strong> contributed to:
+            <strong>{{ person.name || person.login }}</strong> contributed to:
```
Comment (agarny) [inline src/components/GithubContributors.vue]: Much better indeed!

**Review by agarny — COMMENTED** [review]
Just a few more minor issues.

File: src/components/AboutContent.vue (hunk 2 above)
Comment (hsorby) [reply src/components/AboutContent.vue:10]: Can't it is decided by eslint.

Comment (agarny) [reply src/components/AboutContent.vue:10]: Ok, annoying, but fair enough.

File: src/components/AboutContent.vue (hunk 3 above)
Comment (hsorby) [reply src/components/AboutContent.vue]: For me it is important so visitors don't come to any preconceived notion on importance due to the order of the names.

Comment (agarny) [reply src/components/AboutContent.vue]: Are there not more important things in life?

**Review by agarny — APPROVED** [review]

**Pull request merged**